### DISCUSSION TITLE
A Discord Server for the Community

### DISCUSSION THREAD

**Discussion by AmibeSkyfy16** [remkop/picocli#1662] [Q&A] [answered]
Any plan about a Discord server around Picocli, with channel like announcements, java-help, kotlin-help, tips,  etc ?

**Comment by remkop** [comment, marked as answer]
Hi @AmibeSkyfy16, I have no plans to set up a Discord space at the moment. :sweat_smile: 
My time to spend on picocli is quite limited recently, and another consideration is that
it is convenient for me if all questions/discussions happen in one place (GitHub) so I don't have to visit multiple sites to keep track.
Hope you enjoy using picocli! :wink:
